<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maquette</class>
 <widget class="QMainWindow" name="maquette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartStation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_supprimer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1141</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des emplacements</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>431</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout un emplacement</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Localisation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre maximale</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_localisation">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nb_max">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>431</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Suppression</string>
      </property>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sup">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_id_afficher">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>661</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Afficher les Emplacements :</string>
      </property>
      <widget class="QTableView" name="tab_emplacement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>491</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_afficher">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_triId">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>120</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri par ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_triLocalisation">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>150</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri par localisation</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_triPrix">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>210</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri par prix</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_triNbMax">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>180</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri par nombre max</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>340</y>
        <width>631</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Calculer le budget</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>La somme des prix:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_calculer">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>50</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Calculer</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="res_calculer">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Localisation</string>
     </attribute>
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>1031</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>431</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Localisation:</string>
      </property>
      <widget class="QComboBox" name="Localisation_combobox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_localisation">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Localisation</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_localisation_afficher">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Localisation afficher</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QFrame" name="horizontalFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1091</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_stat">
         <property name="text">
          <string>STATISIQUE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <widget class="QPushButton" name="pb_option">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>491</width>
        <height>391</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>489</width>
         <height>389</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Arduino</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>781</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>L'entrée d'une voiture</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Voiture</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_entrer">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>entrer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>280</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat barrière : </string>
       </property>
      </widget>
      <widget class="QLabel" name="etat_label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>280</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
      <widget class="QLabel" name="etat_label_entrer">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>110</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
